packages/esm-patient-chart-app/src/banner/contact-details/contact-details.component.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { InlineLoading } from '@carbon/react';
+import { useRelationships } from './relationships.resource';
+import { usePatientContactAttributes } from '../hooks/usePatientAttributes';
+import styles from './contact-details.scss';
+
+interface ContactDetailsProps {
+  address: Array<fhir.Address>;
+  telecom: Array<fhir.ContactPoint>;
+  patientId: string;
+}
+
+const Address: React.FC<{ address?: fhir.Address }> = ({ address }) => {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <p className={styles.heading}>{t('address', 'Address')}</p>
+      <ul>
+        {address ? (
+          <>
+            <li>{address.postalCode}</li>
+            <li>{address.city}</li>
+            <li>{address.state}</li>
+            <li>{address.country}</li>
```

Comment:
TODO: These fields and their order should be loaded from config or Address Template XML
(Not your worry Hadijah)
New ticket -> https://issues.openmrs.org/browse/O3-1491

Reply:
I will follow up with the ticket above.

Reply:
Oh! Sorry I just made a note of these so we do not lose track of them. You don't have to focus on them now